routes/common/postgres_helper.rb

```diff
@@ -259,10 +259,12 @@ def failover
   end
 
   def send_notification_mail_to_partners(resource, user_email)
-    if resource.flavor == PostgresResource::Flavor::PARADEDB
-      Util.send_email(Config.postgres_paradedb_notification_email, "New ParadeDB Postgres database has been created.",
-        greeting: "Hello ParadeDB team,",
-        body: ["New ParadeDB Postgres database has been created.",
+    if [PostgresResource::Flavor::PARADEDB, PostgresResource::Flavor::LANTERN].include?(resource.flavor)
```

Comment:
maybe
```suggestion
    unless resource.flavor == PostgresResource::Flavor::STANDARD
```

Reply:
I wanted this list to be explicit, so that if/when we add a new flavor in the future, we wouldn't automatically try to send a mail.